packages/webapp/pages/squads/create.tsx

```diff
@@ -137,18 +136,17 @@ function CreatePost(): ReactElement {
           onActiveChange={(active) => setDisplay(active)}
           controlledActive={display}
           shouldMountInactive={false}
-          className={{ header: 'px-1' }}
```

Comment:
Why did we end up removing this header padding?

Reply:
This should be by mistake